pkg/annotation/annotation.go

```diff
@@ -0,0 +1,5 @@
+package annotation
+
+const (
+	InstanceID = "aws-operator.giantswarm.io/instance"
+)
```

Comment:
My general impression of the `pkg` is this is kind of code that is "ready" to be used outside. I tend to think of it as a code being a candidate for extraction to a separate library. I'm not certain if this would be useful outside aws-operator. However I don't want to block this if this makes you happy.

Reply:
I like the splitting out of annotations. I'm not deep enough into Node Pools to know if this here needs be shared.

But we do need this in `app-operator` and `chart-operator`. We have an annotation for `chart-operator.giantswarm.io/allow-force-upgrade`. We also need `app-operator.giantswarm.io/allow-force-upgrade` and we need to pass the value from the app CR on to the chart CR.